aidants_connect_habilitation/templates/issuer_space.html

```diff
@@ -8,94 +8,119 @@
-                  <a href="{% url "habilitation_validation" issuer_id=issuer.issuer_id uuid=organisation.uuid %}"
-                     class="button primary">Soumettre la demande</a>
+    <div class="fr-grid-row fr-grid-row--gutters fr-grid-row--middle flex-between fr-mb-3w">
+      <div class="">
+        <h2 class="fr-h6 fr-mb-0">Vos demandes d'habilitation</h2>
+        <p>Pour rappel, vous devez déposer une demande d'habilitation par lieu d'accueil.</p>
+      </div>
+      <div class="">
+        <a href="{% url 'habilitation_new_organisation' issuer_id=issuer.issuer_id %}" class="fr-btn fr-btn--icon-left fr-icon-add-line">
+          Déposer une nouvelle demande
+        </a>
+      </div>
+    </div>
+    <div class="fr-grid-row fr-grid-row--gutters fr-mb-6w">
+      <div class="fr-table--lg fr-table fr-table width-100" id="table-sm-component">
+        <div class="fr-table__wrapper">
+          <div class="fr-table__container">
+            <div class="fr-table__content">
+              <table class="with-valid-mandate">
+                <caption class="sr-only">Table des demandes d'habilitation</caption>
+                <thead>
+                  <tr>
+                    <th scope="col">Date</th>
+                    <th scope="col">Statut de la demande</th>
+                    <th scope="col">Lieu d'accueil</th>
+                    <th scope="col">Adresse</th>
+                    <th scope="col">Nb d'aidants</th>
+                    <th scope="col">Actions</th>
+                  </tr>
+                </thead>
+                <tbody>
+                {% if issuer.organisation_requests.all|length > 0 %}
+                  {% for organisation in issuer.organisation_requests.all %}
+                    <tr>
+                      <td>{{ organisation.created_at|date:"d/m/Y" }}</td>
+                      <td>
+                        {% if organisation.status == "NEW" %}
+                          <p class="fr-badge fr-badge--new">Brouillon</p>
+                        {% elif organisation.status == "AC_VALIDATION_PROCESSING" %}
+                          <p class="fr-badge fr-badge--info">En attente</p>
+                        {% elif organisation.status == "VALIDATED" %}
+                          <p class="fr-badge fr-badge--success">Éligibilité validée</p>
+                        {% elif organisation.status == "REFUSED" %}
+                          <p class="fr-badge fr-badge--error">Éligibilité refusée</p>
+                        {% elif organisation.status == "CLOSED" %}
+                          <p class="fr-badge disabled">Cloturée</p>
+                        {% elif organisation.status == "CHANGES_REQUIRED" %}
+                          <p class="fr-badge fr-badge--warning">modifications demandées</p>
+                        {% elif organisation.status == "CHANGES_PROPOSED" %}
+                          <p class="fr-badge fr-badge--new">modifications proposées</p>
+                        {% endif %}
```

Comment:
```suggestion
                        <p class="fr-badge fr-badge--new">{{ organisation.get_status_display }}</p>
```

[`Model.get_FOO_display`](https://docs.djangoproject.com/en/5.1/ref/models/instances/#django.db.models.Model.get_FOO_display)